### PR TITLE
Added Landing page

### LINKED ISSUE #16: Landing Page for the project

### PR DESCRIPTION
**Pull request by romabulani**
- Landing page with categories
- User can choose the category and navigate to Explore Page

### CODE REVIEW

File: src/components/landing-page/landingpage.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+}
+
+.hero-para {
+  font-size: 1.8rem;
+}
+
+.hero-text {
+  align-items: center;
+  margin: 10px;
+  row-gap: 10px;
```
Comment (Rohanmond) [inline src/components/landing-page/landingpage.css:25]: better to maintain one unit

**Review by Rohanmond — APPROVED** [review]
code is looking good to merge

**Review by anuj-kumary — APPROVED** [review]
Nice UI also code is very clean and good use of naming conventions.
Good to merge.

**Review by OmKakatkar — APPROVED** [review]
Like:
* The use of barrel imports
* Use of meaningful variable names

File: src/components/landing-page/landingpage.css (hunk 1 above)
Comment (romabulani) [reply src/components/landing-page/landingpage.css:25]: I am using rem above 12px.

**Pull request merged**